src/deepsparse/v2/schedulers/continuous_batching_scheduler.py

```diff
@@ -161,8 +165,18 @@ def add_engine_operator(
         for batch_size in batch_sizes:
             if batch_size == 1:
                 continue  # already added
-            operator_engines[batch_size] = operator_engines.create_engine(
-                batch_size=batch_size
+
+            override_model_path = None
+            # text generation/NLEngineOperator specific; could add generic method
+            # for all engine_operators, if desired
+            if hasattr(engine_operator, "override_model"):
+                override_model_path, _, _ = engine_operator.override_model(
+                    model_path=engine_operator.model_path, batch_size=batch_size
+                )
+
+            # will break for internal kv_cache; needs additional argument
```

Comment:
I propose for now just raising `NotImplementedError`, we can figure it out later.

Reply:
this case wont ever run for internal kv cache